packages/web-api/src/methods.ts

```diff
@@ -979,6 +988,17 @@ export interface AdminUsersSessionInvalidateArguments extends WebAPICallOptions,
   session_id: string;
   team_id: string;
 }
+export interface AdminUsersSessionGetSettingsArguments extends WebAPICallOptions, TokenOverridable {
+  user_ids: string[];
+}
+export interface AdminUsersSessionSetSettingsArguments extends WebAPICallOptions, TokenOverridable {
+  user_ids: string[];
+  desktop_app_browser_quit?: boolean;
+  duration?: number;
```

Comment:
❓ Just for my own understanding, how is the argument `duration` mapped to [`durationOf`](https://api.slack.com/methods/admin.users.session.getSettings#response)?

Reply:
@mwbrooks The document was wrong. I've already sent a fix to the document team.